numerics/hermite3.hpp

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <set>
+#include <utility>
+
+#include "quantities/quantities.hpp"
+
+namespace principia {
+
+using quantities::Difference;
+using quantities::Exponentiation;
+using quantities::Quotient;
+
+namespace numerics {
+
+// A 3rd-degree Hermite polynomial defined by its values and derivatives at the
+// bounds of some interval.
+template<typename Argument, typename Value>
+class Hermite3 {
+ public:
+  using Derivative = Quotient<Difference<Value>, Difference<Argument>>;
```

Comment:
in named_quantities,
```cpp
template<typename Argument, typename Value>
using Derivative = Quotient<Difference<Value>, Difference<Argument>>;
```
change `Variation` to something like
```cpp
template<typename T>
using Variation = Derivative<T, Time>;
```
here,
```cpp
using Derivative = Derivative<Value, Argument>;
```
or make this an issue.